x/ccv/consumer/keeper/distribution_test.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	testkeeper "github.com/cosmos/interchain-security/v3/testutil/keeper"
 	"github.com/cosmos/interchain-security/v3/x/ccv/consumer/types"
+	ccvtypes "github.com/cosmos/interchain-security/v3/x/ccv/types"
```

Comment:
Similarly to my previous [comment](https://github.com/cosmos/interchain-security/pull/1071/files#r1299958690) there seems the naming is consistent between `distribution.go` and `distribution_test.go`.

Reply:
See above